db/Dockerfile

```
@@ -18,7 +18,7 @@ RUN cpan TAP::Parser::SourceHandler::pgTAP
 COPY dbmate /usr/local/bin/dbmate
 RUN set -x && \
     DBMATE="/usr/local/bin/_dbmate" && \
-    ARCH="$(a="$(uname -m)"; [ "$a" = "aarch64" ] && echo "amd64" || echo "$a")" && \
+    ARCH="$(a="$(uname -m)"; [ "$a" = "aarch64" ] && echo "arm64" || [ "$a" = "x86_64" ] && echo "amd64" || echo "$a")" && \
```

Maybe for clarity we rewrite this on multiple lines?

```suggestion
    ARCH="$(uname -m)" && \
    { [ "$ARCH" != "aarch64" ] || ARCH=arm64; } && \
    { [ "$ARCH" != "x86_64" ] || ARCH=amd64; } && \
```